samples/net/sockets/big_http_download/README.rst

```diff
@@ -0,0 +1,84 @@
+.. _sockets-big-http-download:
+
+Socket Big HTTP Download Example
+################################
+
+Overview
+********
+
+The sockets/big_http_download sample application for Zephyr implements
+a more or less simple HTTP GET client using a BSD Sockets compatible API.
```

Comment:
let's delete the "more or less":

    implements a simple HTTP GET client

Reply:
> let's delete the "more or less":

The reason I wrote it like that is because there's already the http_get sample, and that's branded as "simple HTTP client", this sample is a bit more elaborated than that ;-).

Reply:
fixed